addons/account_debit_note/models/account_move.py

```diff
@@ -28,3 +29,16 @@ def action_view_debit_notes(self):
             'view_mode': 'tree,form',
             'domain': [('debit_origin_id', '=', self.id)],
         }
+
+    def action_debit_note(self):
+        self.ensure_one()
+
+        if self.debit_origin_id:
```

Comment:
This could be done in the default_get of the wizard instead.  (These validations make sense, but we should only have them once in the code)

Reply:
Makes more sense actually